src/infrastructure/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Modal } from '@material-ui/core';
+import styles from './Modal.module.scss';
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: '30rem',
+  bgcolor: 'background.paper',
+  boxShadow: 24,
+  p: 4,
+};
+
+export const BaseModal = ({ open, handleClose, children }: ModalProps) => (
+  <Modal open={open} onClose={handleClose}>
+    <Box sx={style}>
```

Comment:
Can't you style this box like @ImanolH96 suggested you now?

Reply:
Sure, I missed that comment.